src/symbolic_value.ml

```diff
@@ -434,6 +432,11 @@ module F32 = struct
   let of_bits x = cvtop ty Reinterpret_int x
 
   let to_bits x = cvtop (Ty_bitv S32) Reinterpret_float x
+
+  let copy_sign x y =
+    let xi = to_bits (abs x) in
+    let yi = to_bits y in
+    of_bits @@ I32.logor xi (I32.logand yi (const_i32 Int32.min_int))
```

Comment:
These implementations seem correct but will likely not be so efficient for the SMT solver.
Ultimately, will want to deal with copy sign in interpret.ml using the Choice monad available there.
However, we may still want to keep the more efficient bit-level impl for the concrete one, so we'll need to add a mechanism for optional implementations in the value interface.